blockchain/chaingen/generator.go

```diff
@@ -535,12 +535,12 @@ func isRevocationTx(tx *wire.MsgTx) bool {
 }
 
 // voteBlockScript returns a standard provably-pruneable OP_RETURN script
-// suitable for use in a vote tx (ssgen) given the block to vote on.
-func voteBlockScript(parentBlock *wire.MsgBlock) []byte {
+// suitable for use in a vote tx (ssgen) given the block hash and height to vote
+// on.
+func voteCommitmentScript(hash chainhash.Hash, height uint32) []byte {
 	var data [36]byte
```

Comment:
magic number

Reply:
This is test code that is intentionally ensuring that any changes to consensus are detected and thus it generates the vote commitment script exactly as required by consensus.  We could certainly define a constant for it, but given this is the only place it's used, I'm not sure it would be an improvement.

Reply:
yep, a comment is fine.  a constant or comment fixes magic numbers :)

Reply:
Comment added.